src/components/collection.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Box } from '@mui/material';
+import { useUser } from '@auth0/nextjs-auth0/client';
+
+const Collection = () => {
+
+  const { user, error, isLoading } = useUser();
+
+  // Make a loading component later to make this more interesting
+  if (isLoading) return <div>Loading...</div>
+  if (error) return <div>{error.message}</div>
```

Comment:
I don't think you should show an error on this component for a general authentication error